package.json

```diff
@@ -17,7 +17,7 @@
     "type-check": "tsc --noEmit",
     "deploy:s3": "bash scripts/deploy-s3.sh",
     "postinstall": "husky install",
-    "codegen:apollo": "apollo client:codegen --endpoint=https://v4.subgraph.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph --target typescript --tsFileExtension=d.ts --outputFlat src/@types/subgraph/",
+    "codegen:apollo": "apollo client:codegen --endpoint=https://v4.subgraph.ropsten.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph --target typescript --tsFileExtension=d.ts --outputFlat src/@types/subgraph/",
```

Comment:
why changing the endpoint here?

Reply:
because this morning rinkeby wasn't working, so we ended up with some errors during build. This should really impact anything, and also rinkeby will be deprecated at some point. But after the fixes on the rinkeby subgraph we should be good to go back to it if we want to.